src/main/java/org/jenkinsci/plugins/workflow/support/pickles/ExecutorPickle.java

```diff
@@ -93,8 +112,21 @@ protected Executor tryResolve() throws Exception {
                 Future<Queue.Executable> future = item.getFuture().getStartCondition();
 
                 if (!future.isDone()) {
-                    // TODO JENKINS-26130 we might be able to detect that the item is blocked on an agent which has been deleted (not just offline), and abort ourselves
-                    LOGGER.log(Level.FINER, "{0} not yet started", item);
+                    Queue.Task task = item.task;
+                    Label nodeNameLabel = task.getAssignedLabel();  // Once execution started, this was set to the Computer name
```

Comment:
Hmm. That is true of `PlaceholderTask`. But `ExecutorPickle` was originally designed to potentially support other `Executor`s. So it would be better to do the `instanceof` check to begin with, and for any other kind of `Task`, just keep the original behavior.